code/addons/docs/src/blocks/blocks/DocsContainer.tsx

```
@@ -28,13 +28,18 @@ export const DocsContainer: FC<PropsWithChildren<DocsContainerProps>> = ({
 }) => {
   const slugger = useMemo(() => createDocsSlugger(), []);
   let toc;
+  // Language of docs prose (descriptions, ArgTypes description cells, free MDX prose).
+  let lang;
```

Biome (2.5.1)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)